common/app/common/Edition.scala

```diff
@@ -104,10 +107,10 @@ object Edition {
     val path = request.path
     path match {
       case EditionalisedId(editionId, section) if Edition.defaultEdition.isEditionalised(section.drop(1)) =>
-        val links = Edition.all.map(EditionLink(_, section))
+        val links = Edition.editionsByRequest(request).map(EditionLink(_, section))
         links.filter(link => link.edition.isEditionalised(link.path.drop(1)))
       case EditionalisedFront(_) =>
-        all.map(EditionLink(_, "/"))
+        editionsByRequest(request).map(EditionLink(_, "/"))
```

Comment:
There's an interesting consequence here for the [`<link rel="alternate" >`](https://developers.google.com/search/docs/advanced/crawling/localized-versions#html) links generated by this bit of code - they look like this in the HTML source of our pages (eg https://www.theguardian.com/us):

```
<link rel="alternate" href="https://www.theguardian.com/uk" hreflang="en-GB" />
<link rel="alternate" href="https://www.theguardian.com/us" hreflang="en-US" />
<link rel="alternate" href="https://www.theguardian.com/au" hreflang="en-AU" />
<link rel="alternate" href="https://www.theguardian.com/international" hreflang="en" />
```

If Europe is added, we would have **two** lines with `hreflang="en"` - which might conceivably create SEO errors in the Google crawler (stuff like https://search.google.com/search-console/r/unknown-type?resource_id=https%3A%2F%2Fwww.theguardian.com%2F)

Reply:
Having two editions with the same `hreflang` value does in fact break the alternate URLs: 
<img width="1476" alt="Screenshot 2022-09-08 at 16 22 55" src="https://user-images.githubusercontent.com/74187452/189169597-ae3352f6-9f3d-4fea-b5f6-f295fe8aaf2a.png">

On solution is setting the International Edition `hreflang` as `x-default` which is usually used for international landing pages, but that might have SEO implications that need to be discussed with digital editors.
<img width="1486" alt="Screenshot 2022-09-08 at 15 59 44" src="https://user-images.githubusercontent.com/74187452/189169876-306f24b6-c369-4f3b-acae-8e8fa01e27f4.png">

Reply:
We had a conversation with Peter Martin (digital editor) today and we think we want to exclude Europe from alternate pages for now unless it proves necessary/beneficial to have it in the future in which case we will revisit this. We decided agains using "x-default" as it will have some unwanted side-effects such as redirecting European traffic to International if they were using any language other than "en".